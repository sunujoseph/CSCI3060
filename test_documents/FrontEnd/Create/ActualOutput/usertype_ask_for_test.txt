path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Create\ActualDTF\usertype_ask_for_test.txt
Enter Username: temp: |
create
neÍÍÍÍÍÍÍııııİİİr*^t5ı|
username: |admin1
create
n| not found
Error: Not Logged In
Error: Not Logged In
