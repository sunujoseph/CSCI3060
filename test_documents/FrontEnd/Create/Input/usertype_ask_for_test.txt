login
admin1
create
new_user
