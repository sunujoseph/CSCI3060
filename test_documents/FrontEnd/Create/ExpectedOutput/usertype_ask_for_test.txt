Enter Username: 
Enter Username For New User: 
Enter Type For New User: 